src/environment.ts

```diff
@@ -20,12 +20,13 @@ function installLazyGlobals(env: SecureEnvironment, baseGlobalThis: object, base
             const { set: originalSetter, get: originalGetter, value: originalValue } = descriptor;
             if (!isUndefined(originalGetter)) {
                 descriptor.get = function get(): any {
-                    return env.getSecureValue(originalGetter.call(baseGlobalThis));
+                    const value: any = apply(originalGetter, baseGlobalThis, []);
```

Comment:
☝️ TS noob. I'm guessing there isn't a need to specify `any` there since it's inferred as such by default?

Reply:
it should probably be `RawValue` here since this is happening in the outer realm, and will be passed into `getSecureValue` which expects `RawValue`. Jejejeje, yes, it is any anyhow... but eventually we can narrow that down more.